### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Bondok6**
## Hello there 👋
Thanks for taking the time to review the project. 
<hr>

### Requirements 🧾

- Implemented a function for adding a new task (add a new element to the array) ✅
- Implemented a function for deleting a task (remove an element from the array) ✅
- Implemented a function for editing task descriptions ✅
- All changes to the To-Do List  saved in local storage ✅
- My app looks similar to a clone of the part of the minimalist project captured in the video. ✅

### CODE REVIEW

File: src/modules/addTask.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import optionIconUrl from '../imgs/optionsIcon.svg';
+import trashIcon from '../imgs/trashIcon.svg';
+import getData from './arrangeIndex.js';
+
+const input = document.getElementById('task-input');
+const inputItem = document.getElementById('input-item');
+
+getData();
+let id;
+id = Tasks.tasks.length;
```
Comment (ZeenatLawal) [inline src/modules/addTask.js]: - [x] The index of each task should be set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5). To do this your first task should have an index of 1, not 0 kindly fix this.

File: src/modules/allOptions.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+const options = (e) => {
+  // Delete Task
+  const targetId = +e.target.parentNode.parentNode.className;
+  if (e.target.classList.contains('delete-icon')) {
+    const target = e.target.parentNode.parentNode;
+    Tasks.tasks = Tasks.tasks.filter((task) => task.index !== targetId);
+    Tasks.storage(Tasks.tasks);
+    target.remove();
+  }
```
Comment (ZeenatLawal) [inline src/modules/allOptions.js]: - [x] After deleting a task the index of the remaining tasks should be updated so they represent the current list order and are unique as stated in the requirements. The delete function is currently not doing that, kindly modify it.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @Bondok6, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @Bondok6, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

You did a great job working on the requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**